### PR TITLE
Write API key to disk

### PR DESCRIPTION
**Pull request by Jin1L**

### CODE REVIEW

File: LOADERS/CLOUD_DATABASE/LOADER/LOADER.py
Comment (itsjoeoui) [inline LOADERS/CLOUD_DATABASE/LOADER/LOADER.py:71]: Please use the `with` statement to open the file, which will be automatically recycled.

**Comment by smahmed776** [comment]
@Jin1L @jackparmer asked me to write getter and setter method in flojoy utils so that they can be imported from flojoy package and used by Django server. I'm done with that just waiting for approval. So the getter and setter method in LOADER node is redundant IMO.

**Comment by Jin1L** [comment]
> Hey @Jin1L, @jackparmer asked me to write getter and setter method in flojoy utils so that they can be imported from flojoy package and used by Django server. I'm done with that just waiting for approval. So the getter and setter method in LOADER node is redundant IMO.

Hey @smahmed776, thanks for letting us know! Could you ping us on Slack when it is approved?

**Comment by smahmed776** [comment]
> > Hey @Jin1L, @jackparmer asked me to write getter and setter method in flojoy utils so that they can be imported from flojoy package and used by Django server. I'm done with that just waiting for approval. So the getter and setter method in LOADER node is redundant IMO.
> 
> Hey @smahmed776, thanks for letting us know! Could you ping us on Slack when it is approved?

Yes, I'll let you know once it is approved!

File: LOADERS/CLOUD_DATABASE/LOADER/LOADER.py
Comment (smahmed776) [inline LOADERS/CLOUD_DATABASE/LOADER/LOADER.py:23]: The return value of `get_frontier_api_key()` would be either a string or None, so we need to change this checking to `if api_key is not None and ....`

File: LOADERS/CLOUD_DATABASE/LOADER/LOADER.py
Comment (smahmed776) [inline LOADERS/CLOUD_DATABASE/LOADER/LOADER.py:39]: Same here `if api_key is None`

File: LOADERS/CLOUD_DATABASE/LOADER/__init__.py
Comment (smahmed776) [inline LOADERS/CLOUD_DATABASE/LOADER/__init__.py:1]: I don't think these `__init__.py` files are necessary!

**Review by smahmed776 — CHANGES_REQUESTED** [review]

**Review by smahmed776 — APPROVED** [review]
LGTM!

**Pull request merged**
